client/pcr.go

```diff
@@ -33,6 +33,26 @@ func min(a, b int) int {
 	return b
 }
 
+// Get a list of selections corresponding to the TPM's implemented PCRs
+func implementedPCRs(rw io.ReadWriter) ([]tpm2.PCRSelection, error) {
+	caps, moreData, err := tpm2.GetCapability(rw, tpm2.CapabilityPCRs, math.MaxUint32, 0)
+	if err != nil {
+		return nil, fmt.Errorf("listing implemented PCR banks: %w", err)
+	}
+	if moreData {
```

Comment:
Why moreData results in error here? Is this a temporary solution?

Reply:
`moreData` should never be set for this type of call (per the spec), so erroring out here is fine. I'll add a comment explaining.

Reply:
I see, then we should also make sure the go-tpm library https://github.com/google/go-tpm/blob/7b1a3bb951f3f6af1a0a5d0190278480e522d752/tpm2/tpm2.go#L288 won't return that